packages/shared/src/components/profile/devcard/DevCard.tsx

```diff
@@ -71,24 +71,24 @@ enum DevCardTheme {
 }
 
 const themeToLinearGradient = {
-  [DevCardTheme.Default]: `linear-gradient(0deg, #FFFFFF, #FFFFFF), linear-gradient(152.83deg, #FFFFFF 51.92%, ${colors.salt['20']} 85.8%)`,
+  [DevCardTheme.Default]: `linear-gradient(0deg, ${colors.salt['0']}, ${colors.salt['0']}), linear-gradient(152.83deg, ${colors.salt['0']} 51.92%, ${colors.salt['20']} 85.8%)`,
   [DevCardTheme.Iron]:
-    `linear-gradient(135.48deg, ${colors.pepper['10']} 0%, ${colors.pepper['90']} 20%, ${colors.pepper['10']} 47.5%, #2C303A 67%, ${colors.pepper['90']} 83%, ${colors.pepper['10']} 100%),\n` +
+    `linear-gradient(135.48deg, ${colors.pepper['10']} 0%, ${colors.pepper['90']} 20%, ${colors.pepper['10']} 47.5%, ${colors.pepper['50']} 67%, ${colors.pepper['90']} 83%, ${colors.pepper['10']} 100%),\n` +
     `linear-gradient(180deg, ${colors.salt['20']} 0%, ${colors.salt['10']} 85.42%)`,
   [DevCardTheme.Bronze]:
-    `linear-gradient(135.48deg, #C98463 0%, ${colors.burger['40']} 20%, #FFB760 47.5%, #C98463 67%, ${colors.burger['40']} 83%, #C98463 100%),\n` +
+    `linear-gradient(135.48deg, ${colors.burger['10']} 0%, ${colors.burger['40']} 20%, ${colors.burger['10']} 47.5%, ${colors.burger['10']} 67%, ${colors.burger['40']} 83%, ${colors.burger['10']} 100%),\n` +
```

Comment:
Will tailwind be able to statically analyze this?

Reply:
Yes, however not ideal for the long term, think when we introduce the new motives we change.